Power Up Proto/src/org/usfirst/frc/team5493/robot/Robot.java

```diff
@@ -45,7 +45,7 @@ public void robotInit() {
     	
     	climber = new OneClimbyBoi();
     	cubeControls = new CubeControls();
-    	throwDaggersInBensEyes = new ThrowDaggersInBensEyes();
+//    	throwDaggersInBensEyes = new ThrowDaggersInBensEyes();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)